tools/topology/topology2/production/tplg-targets-hda-generic.cmake

```diff
@@ -6,4 +6,13 @@ DEEPBUFFER_D0I3_COMPATIBLE=true"
 # HDA topology with mixer-based pipelines for HDA and passthrough pipelines for HDMI
 "sof-hda-generic\;sof-hda-generic\;HDA_CONFIG=mix,DEEPBUFFER_FW_DMA_MS=100,\
 DEEPBUFFER_D0I3_COMPATIBLE=true"
+
+# HDA topology with mixer-based pipelines for HDA and
+# passthrough pipelines for HDMI and
+# 2 or 4 DMIC, no NHLT blob included in topology
+"sof-hda-generic\;sof-hda-generic-2ch\;HDA_CONFIG=mix,NUM_DMICS=2,\
+DEEPBUFFER_FW_DMA_MS=100,DEEPBUFFER_D0I3_COMPATIBLE=true"
+"sof-hda-generic\;sof-hda-generic-4ch\;HDA_CONFIG=mix,NUM_DMICS=4,\
```

Comment:
[8:49 PM] Sridharan, Ranjani
when you have NUM_DMICS as 4 you also need PDM1_MIC_A_ENABLE=1,PDM1_MIC_B_ENABLE=1

Reply:
Please correct me but I think it is there... I copied it from 65e4c1f8b5192f0ee